appveyor.yml

```diff
@@ -107,7 +107,7 @@ artifacts:
     name: Rubberduck
 deploy:
   - provider: GitHub
-    tag: Rubberduck Stable v$(appveyor_build_version)
+    tag: RETAGMEWITHAMESSAGE
```

Comment:
i feel that this needs a bit explanation - is this a special code to AV to retag it using the commit message? or is it just a comment to us meatbags to retag it?

Reply:
The latter